waspc/src/Wasp/Common.hs

```diff
@@ -13,3 +16,12 @@ data DbMigrationsDir
 
 dbMigrationsDirInWaspProjectDir :: Path' (Rel WaspProjectDir) (Dir DbMigrationsDir)
 dbMigrationsDirInWaspProjectDir = [reldir|migrations|]
+
+data StartOptions = StartOptions
+  { webClientPort :: Maybe String,
```

Comment:
Ideally these would already be parsed somewhere earlier and stored here as a number! Probably Int.

Reply:
Yep, done.